### PR TITLE
Added GameManager Class and Placeholder bgm

### PR DESCRIPTION
**Pull request by Maxroo**
can add some need to shared variable into gamemanager class.

### CODE REVIEW

File: src/main/java/ca/ciccc/silverBullet/controller/GameManger.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ca.ciccc.silverBullet.controller;
+
+import FileInput.FileInput;
+import javafx.scene.media.AudioClip;
+
+public class GameManger {
```
Comment (nator333) [inline src/main/java/ca/ciccc/silverBullet/controller/GameManger.java]: Typo

Comment (nator333) [reply src/main/java/ca/ciccc/silverBullet/controller/GameManger.java]: I suppose you should change this signature to AbstractController class.
Then move this class into the controller package.

File: src/main/java/ca/ciccc/silverBullet/SilverBulletApp.java [hunk 2]
```diff
@@ -1,9 +1,11 @@
 package ca.ciccc.silverBullet;
 
+import FileInput.FileInput;
```
Comment (nator333) [inline src/main/java/ca/ciccc/silverBullet/SilverBulletApp.java]: Can you include the word Media into this FileInput class?

Comment (nator333) [reply src/main/java/ca/ciccc/silverBullet/SilverBulletApp.java]: And please change its package as well

Comment (Maxroo) [reply src/main/java/ca/ciccc/silverBullet/SilverBulletApp.java]: then they have to extend fileinput class to use the sound.

**Comment by nator333** [comment]
@Maxroo 
Resolve conflicts

**Pull request merged**